timemachine/ff/handlers/utils.py

```diff
@@ -11,6 +26,8 @@ def sort_tuple(arr):
         return arr
     elif arr[0] > arr[-1]:
         return container_type(reversed(arr))
+    elif arr[0] == arr[-1]:
+        raise Exception("arr[0] == [-1]")
```

Comment:
nit: maybe something like
```suggestion
        raise ValueError("Invalid bond with first and last indices equal")
```

Reply:
committed inline - thanks